### ISSUE TITLE
Ensure heading level matches the heading's visual importance/level

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#2141]
Researching whether the visual heading level not matching the semantic heading level is a 1.3.1 Info & relationships fail. 

The Level Access interpretation I found online states '_The heading element level must match the level communicated visually by the headings on the page_.' I assume that means h1 should be visually bigger than h2, h3 etc. and that this should be reflected in the heading level markup.

I'm finding it difficult to phrase this question, so, hopefully I am making some sense.

Reference:
[Ensure heading level matches the heading's visual importance/level](https://amp.levelaccess.net/public/standards/view_best_practice.php?violation_id=387)

**Comment by JAWS-test** [comment]
The level of a headline results more from content criteria and not necessarily from visual ones. I.e. a designer may decide to make the main heading (<h1>) visually smaller than a section heading (<h2>). The important thing is that the programmatic heading level matches the content level. In most cases, this also matches the visual level.

**Comment by alastc** [comment]
I think [David Swallow's article](https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag/) is probably the best all-round read on pass/fails for headings.

The bit he has under "Illogical heading order" is about this issue, although I'd probably describe more like this bit: "A heading that’s visually less important must be assigned a heading that’s lower in the hierarchy."

Size is one factor, but you could have a smaller heading that is given greater prominence by location and styling.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
